src/App.jsx

```diff
@@ -14,33 +15,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortBy(status, arr) {
+  const sortedArr = [...arr];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (status) {
+    case 'by-alphabetically':
```

Comment:
The case label 'by-alphabetically' should be 'alphabetically' to match the argument passed in the `handleSort` function on line 69. This mismatch will cause the alphabetical sorting not to work as expected.